viadot/tasks/git.py

```diff
@@ -0,0 +1,96 @@
+import shutil
+from typing import Dict
+
+import pygit2
+from prefect.utilities.logging import get_logger
+from prefect.utilities.tasks import defaults_from_attrs
+
+from viadot.task_utils import *
```

Comment:
Is it possible to avoid using *?

Reply:
There was an issue with the circular import